docker/cloudsim_sim/run_sim.bash

```diff
@@ -14,4 +14,4 @@ bwm-ng -o csv -c 0 -t 1000 -T rate -I eth0 >> $FILE &
 atop -R -w /tmp/ign/logs/atop_log &
 
 export ROS_LOG_DIR=/tmp/ign/logs/ros
-ign launch -v 4 $@
+ign launch -v 4 $@ | tee /tmp/ign/logs/gzserver.log
```

Comment:
We should pipe `stderr` as well.

```suggestion
ign launch -v 4 $@ |& tee /tmp/ign/logs/gzserver.log
```

Reply:
Do we want to log `stderr` in the `stdout` too?